frontend/src/routes/generate_summary/+page.svelte

```diff
@@ -37,10 +59,10 @@
     <TopBar />
 
     <div class="text-4xl font-inter font-bold mb-4 text-black flex flex-col justify-center items-center mt-20">
```

Comment:
Please change to use h1/h2/h3/subheading tag. Remove redundant styling. refer to app.css for what has already been applied